docs/source/workflow_outputs.rst

```diff
@@ -321,14 +323,17 @@ Annual unmet hours are listed below.
   =========================  =====
   Unmet Hours: Heating (hr)  Number of hours where the heating setpoint is not maintained.
   Unmet Hours: Cooling (hr)  Number of hours where the cooling setpoint is not maintained.
+  Unmet Hours: Driving (hr)  Number of hours where the driving schedule of electric vehicles is not met
   =========================  =====
 
-These numbers reflect the number of hours during the heating/cooling season when the conditioned space temperature deviates more than 0.2 deg-C (0.36 deg-F) from the heating/cooling setpoint.
+The unmet heating and cooling numbers reflect the number of hours during the heating/cooling season when the conditioned space temperature deviates more than 0.2 deg-C (0.36 deg-F) from the heating/cooling setpoint.
+The unmet driving number represents the total time in which the charging schedule fails to meet the demand from the driving schedule. This includes periods when the discharge schedule exceeds zero while the EV battery's state of charge is at its minimum level. Unmet driving hours indicate unrealized discharging events, resulting in reduced charging energy.
```

Comment:
I'm still a bit unclear on whether this is hours of unmet discharging or charging (or both). The first sentence says "represents the total time in which the **charging** schedule fails to meet the demand" but the second sentence says it "includes periods when the **discharge** schedule exceeds zero while the EV battery's state of charge is at its minimum level".

I also would prefer a clearer output name than "Driving". Ideally it would be "EV Charging", but like I said above, I'm not actually sure if that's what it is. "EV Driving" would also be fine.

Reply:
Discussed with @rajeee and we are updating the language a bit in the next commit.